pkg/pb/metadata/metadata.proto

```diff
@@ -71,11 +71,16 @@ message DeleteFileRequest {
 
 message DeleteFileResponse { }
 
+message ForceGarbageCollectionRequest { }
+
+message ForceGarbageCollectionResponse { }
+
 service MetadataService { 
     rpc GetFile (GetFileRequest) returns (GetFileResponse);
     rpc PutFile (PutFileRequest) returns (PutFileResponse);
     rpc DeleteFile (DeleteFileRequest) returns (DeleteFileResponse);
     rpc GetMetadata (GetMetadataRequest) returns (GetMetadataResponse);
     rpc SetMetadata (SetMetadataRequest) returns (SetMetadataResponse);
     rpc GetDirectoryKeys (GetDirectoryKeysRequest) returns (GetDirectoryKeysResponse);
+    rpc ForceGarbageCollection (ForceGarbageCollectionRequest) returns (ForceGarbageCollectionResponse);
```

Comment:
This should not be in the public API, we're only using this to test. Here's what to do instead:

```go
// Within cmd/metadata-server/server.go

// To only be used for testing purposes.
func TestForceGC(s *Server) error {
	return s.runGarbageCollection(context.Background())
}
```

Now you no longer need the proto changes.